<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1574834797256' stop='1574834812695'>
    <name>demo calculator tests</name>
    <title>demo calculator tests</title>
    <test-cases>
        <test-case start='1574834797257' status='passed' stop='1574834800686'>
            <name>addition test</name>
            <title>addition test</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1574834800686' status='failed' stop='1574834803474'>
            <name>subtraction test</name>
            <title>subtraction test</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
            <failure>
                <message>Expected '2' to equal '3'.</message>
                <stack-trace>Error: Failed expectation
                    at UserContext.&lt;anonymous> (C:\Users\saide\ProtractorTutorial\ProtractorDemo\tests\calculator.js:39:34)
                    at C:\Users\saide\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:112:25
                    at new ManagedPromise (C:\Users\saide\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:1077:7)
                    at ControlFlow.promise (C:\Users\saide\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2505:12)
                    at schedulerExecute (C:\Users\saide\AppData\Roaming\npm\node_modules\protractor\node_modules\jasminewd2\index.js:95:18)
                    at TaskQueue.execute_ (C:\Users\saide\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3084:14)
                    at TaskQueue.executeNext_ (C:\Users\saide\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:3067:27)
                    at C:\Users\saide\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:2927:27
                    at C:\Users\saide\AppData\Roaming\npm\node_modules\protractor\node_modules\selenium-webdriver\lib\promise.js:668:7</stack-trace>
            </failure>
        </test-case>
        <test-case start='1574834803474' status='passed' stop='1574834806257'>
            <name>multiplication test</name>
            <title>multiplication test</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1574834806257' status='passed' stop='1574834809603'>
            <name>division test</name>
            <title>division test</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
        <test-case start='1574834809603' status='passed' stop='1574834812692'>
            <name>modulo test</name>
            <title>modulo test</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments/>
        </test-case>
    </test-cases>
</ns2:test-suite>